lib/ex_aws/sqs/request.ex

```diff
@@ -0,0 +1,28 @@
+defmodule ExAws.SQS.Request do
+  @moduledoc false
+
+  def request(client, queue_name, action, params \\ []) do
+    {_, http_method} = ExAws.SQS.Impl |> ExAws.Actions.get(action)
+
+    headers = [
+      {"content-type", "application/x-www-form-urlencoded"},
+    ]
+
+    ExAws.Request.request(http_method, client.config |> url(queue_name, params), [], headers, client)
+    |> parse(client.config)
+  end
+
+  def parse({:error, result}, _), do: {:error, result}
+  def parse({:ok, body}, config) do
+    case config[:json_codec].decode(body) do
```

Comment:
I realize you probably just copied and pasted this from another request module, but do note that (as far as I can tell) SQS only returns XML so we don't want to json parse. As of right now I don't have a policy towards the parsing of XML. JSON has a nice one to one mapping with elixir data structures. XML's structure and meta-data can vary a lot more.

I should look over the structures returned by SQS. Maybe it makes sense to parse that XML into an elixir data structure.